app/controllers/mailing_lists_controller.rb

```diff
@@ -66,6 +66,12 @@ def render_entries_json(entries)
                     serializer: MailingListSerializer)].inject(&:merge)
   end
 
+  def assign_attributes
+    super if params[:mailing_list].present?
```

Comment:
Finde ich eher unschön, aber es ist analog gelöst in [PeopleFilterController](https://github.com/hitobito/hitobito/blob/134d9bd7c957eb40e5748bb09cdb360ae0ac2dc5/app/controllers/people_filters_controller.rb#L72). 

Gibt's noch eine elegantere Variante?

Reply:
In eigenem Controller